### ISSUE TITLE
Issue with plot_capacity

### ISSUE THREAD

**Issue by alexsescu** [calliope-project/calliope#189]
## Problem description

 I have problems when plotting solutions. The model comes to a solution and generates the folder results.csv although it gives me error as follows: 
Saving CSV results to directory: results.csv
Saving HTML file with plots to: plots.html

Error in _get_var_data:
too many indices for array

## If reporting an error when running Calliope on the command line, please re-run your command with the ``--debug`` option, e.g.:

Saving CSV results to directory: results.csv
Saving HTML file with plots to: plots.html
Traceback (most recent call last):
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\cli.py", line 83, in format_exceptions
    yield
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\cli.py", line 256, in run
    model.plot.summary(to_file=save_plots)
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\analysis\plotting\plotting.py", line 204, in summary
    return plot_summary(self._model, **kwargs)
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\analysis\plotting\plotting.py", line 45, in plot_summary
    capacity = _plot(*plot_capacity(model, subset=subset), html_only=True)
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\analysis\plotting\capacity.py", line 237, in plot_capacity
    'location_axis': location_axis, 'value_axis': value_axis},
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\analysis\plotting\util.py", line 49, in get_data_layout
    **get_var_data_kwargs)
  File "C:\Program_Files\Anaconda3\envs\calliope\lib\site-packages\calliope\analysis\plotting\capacity.py", line 122, in _get_var_data
    x, y = x[::-1], y[::-1]  # Make sure that sorting is from bottom down

## Steps to reproduce the problem

calliope run model.yaml --save_csv=results.csv --save_plots=plots.html

## Calliope version 

0.6.3

**Comment by sjpfenninger** [comment]
@alexsescu Sounds like there might be an issue in one of the plotting functions. To diagnose this I think we need more information about your model -- are you doing anything special?

**Comment by sjpfenninger** [comment]
Will be addressed in #357

**Comment by jsejdija** [comment]
I encountered the same error in Calliope version 0.6.10. The problem occrus when you run calliope with only one location defined. I fixed it by changing calliope\analysis\plotting\capacity.py", line 136, in _get_var_data
from this

 `x, y = x[::-1], y[::-1] # Make sure that sorting is from bottom down` 

to this

```
try:
    x, y = x[::-1], y[::-1]  # Try sorting from bottom down
except:
    x, y = x, y

```
